core/forms/confirm-reset-password-form.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
```

Comment:
I dont see where you found all this subfolders in the `core` on the diagram.

confirm-reset-password seems to be part of the AUTH module.